force-app/main/default/lwc/staffAvailabilityItem/staffAvailabilityItem.js

```diff
@@ -35,13 +35,14 @@ export default class StaffAvailabilityItem extends LightningElement {
 		notifyViaAlerts: false // Use alerts instead of toast to notify user
 	};
 
-	renderedCallback() {
-
+	@api
+	get assignmentRecord() {
```

Comment:
why not null?

Reply:
could be anything really, just needed a get to create the set piece ^^